code/game/objects/items/weapons/holosign.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/holosign_creator
 	name = "holographic sign projector"
-	desc = "A handy-dandy holographic projector that displays a janitorial sign."
+	desc = "This shouldnt exist, if it does tell a coder"
```

Comment:
```suggestion
	desc = "This shouldnt exist, if it does, tell a coder"
```

Reply:
If a comma goes there, the first comma needs to be a full stop. Otherwise "if it does" becomes a parenthetical statement and that's illegal here. The grammar police are coming